pkg/cmd/cli/cmd/startbuild.go

```diff
@@ -58,11 +64,28 @@ Examples:
 				newBuild = buildutil.GenerateBuildFromBuild(build)
 			}
 
+			var wg sync.WaitGroup
+			if streamLogs {
+				rd, err := client.Get().Namespace(namespace).Prefix("redirect").Resource("buildLogs").Name(buildName).Stream()
```

Comment:
@bparees yes, if you take a look at the code behind `osc build-logs`, it's the same as the methods I have invoked here ie. specify a build or a buildConfig and stream its logs.